loom_js_test/src/challenge-after-demo.ts

```diff
@@ -111,5 +90,11 @@ export async function runChallengeAfterDemo(t: test.Test) {
   // Close the websocket, hacky :/
   // @ts-ignore
   web3.currentProvider.connection.close()
+  // @ts-ignore
+  authority.web3.currentProvider.connection.close()
+  // @ts-ignore
+  dan.web3.currentProvider.connection.close()
+  // @ts-ignore
+  mallory.web3.currentProvider.connection.close()
```

Comment:
Should probably have `User.disconnect()` that does this, or `Entity.disconnect()` whichever makes more sense.